client/src/views/devices/ExportRecoveryDevicePage.vue

```diff
@@ -3,100 +3,142 @@
 <template>
   <ion-page>
     <ion-content :fullscreen="true">
-      <div class="container">
-        <div class="recovery-container">
-          <div v-if="state === ExportDevicePageState.Start">
-            <ms-informative-text>
-              {{ $t('ExportRecoveryDevicePage.subtitles.newPassword') }}
-            </ms-informative-text>
+      <div class="recovery">
+        <!-- STEP 1 • Information -->
+        <div
+          class="recovery-container information"
+          v-if="state === ExportDevicePageState.Start"
+        >
+          <ms-informative-text>
+            {{ $t('ExportRecoveryDevicePage.subtitles.newPassword') }}
+          </ms-informative-text>
```

Comment:
Is there a way to highlight parts of the text with another font color as seen in the mockups ?
<img width="938" alt="Screenshot 2023-12-01 at 15 24 41" src="https://github.com/Scille/parsec-cloud/assets/41511029/08cb33ee-4dac-4583-bdb3-9d56262b367e">

Reply:
We cannot highlight part of the text because these 2 lines are managed by the `informative-text` component for now.